R/check.R

```diff
@@ -35,6 +35,13 @@ pkgdown_sitrep <- function(pkg = ".") {
   cli::cli_rule("Sitrep")
 
   pkg <- as_pkgdown(pkg)
+  if (pkg$bs_version == 3) {
+    cli::cli_inform(c(
```

Comment:
this will be visible in CI logs? should there be more info, a new short blog post maybe?

Reply:
I'm going to think more about how to expose the sitrep more clearly in automated checks.